src/models/applicationState.ts

```diff
@@ -169,7 +170,22 @@ export interface IRegion {
  */
 export interface ITagMetadata {
     name: string;
-    properties: object;
+    properties?: object;
+}
+
+/**
+ * @name - Bouding Box
+ * @description - Defines the tag usage within a bounding box region
+ * @member left - Defines the left x boundary for the start of the bounding box
+ * @member top - Defines the top y boundary for the start of the boudning box
+ * @member width - Defines the width of the bounding box
+ * @member height - Defines the height of the bounding box
+ */
+export interface IBoundingBox {
```

Comment:
It is not confusing/denormalized to have a specific boundingBox property when we already have the points array?  

We should probably just implement an utility function that return an IBoundingBox interface from the points array.  This function should work with all region types.  For polygon eventually just taking the almost left-right-up-down coordinates and create a box with these.

Reply:
@JacopoMangiavacchi I think this warrents a bigger discussion to ensure that our `IRegion` implementation is good enough for all current region types.  Otherwise I'm fine with making this change and is also why I mentioned in another comment that we should have a discussion about this.

Reply:
Will refactor in new PR.